linkerd/examples/consul.yaml

```diff
@@ -1,13 +1,15 @@
 # Do service discovery lookups against the Consul service catalog.
 namers:
 - kind: io.l5d.consul
-  includeTag: true
+  includeTag: false
   useHealthCheck: false
+  host: 0.0.0.0
+  port: 8300
```

Comment:
ah, I did not mean to commit this - I had made some changes locally for testing.